### ISSUE TITLE
When i try to run this plugin it gives me error that Unexpected token

### ISSUE THREAD

**Issue by sanjeevkumar143** [BuilderIO/html-to-figma#29]
Syntax error on line 10: Unexpected token
import { traverseLayers } from "./functions/traverse-layers";

**Comment by steve8708** [comment]
hey @sanjeevkumar143 - can you add more context about how you are running into this issue? Are you running from a browser (if so what build tool - webpack, rollup, parcel, etc), node.js, other?

**Comment by sanjeevkumar143** [comment]
no, I am using the Figma software after a lot of hours wasted I change my tsc.config file and made change "module":"udm" and it work all errors gone but now am facing another issue which is the plugin frozen https://prnt.sc/sus730 it's just loading nothing come up

**Comment by sanjeevkumar143** [comment]
https://prnt.sc/sus7l8 these error comes up in my console

**Comment by sanjeevkumar143** [comment]
nothing come up in Figma console just stuck in loading

**Comment by steve8708** [comment]
oh I see - you are running this locally. have you tried `npm install && npm run dev` from the root of this project before running the plugin in development mode?

**Comment by sanjeevkumar143** [comment]
> 
> 
> oh I see - you are running this locally. have you tried `npm install && npm run dev` from the root of this project before running the plugin in development mode?

yes i tried it
